website/src/controllers/api-v3/groups.js

```diff
@@ -123,6 +123,10 @@ api.getGroups = {
       return previousValue.concat(Array.isArray(currentValue) ? currentValue : [currentValue]); // otherwise concat the new results to the previousValue
     }, []);
 
+    results = _.uniq(results, (group) => {
```

Comment:
Nevermind about this. For some reason the await-to-eventually-have-lengthOf function was not counting the return obejct correctly. I am about to push a revision.